nipy/algorithms/registration/groupwise_registration.py

```diff
@@ -758,19 +741,71 @@ def _init(self, images, tr, slice_times, slice_info, affine_class):
         self._time_interp = time_interp
 
     def estimate(self,
-                 loops=LOOPS,
+                 loops=5,
                  between_loops=None,
                  align_runs=True,
-                 speedup=SPEEDUP,
-                 borders=BORDERS,
-                 optimizer=OPTIMIZER,
+                 speedup=5,
+                 borders=(1, 1, 1),
+                 optimizer='ncg',
                  xtol=XTOL,
                  ftol=FTOL,
                  gtol=GTOL,
                  stepsize=STEPSIZE,
                  maxiter=MAXITER,
                  maxfun=MAXFUN,
-                 refscan=REFSCAN):
+                 refscan=0):
+        """
+        Estimate motion parameters.
+
+        Parameters
+        ----------
+        speedup: int or sequence of ints
+            Determines an isotropic sub-sampling factor, or a sequence
+            of such factors, applied to the scans to perform motion
+            estimation. If a sequence, several estimation passes are
+            applied.
+        loops : int or sequence of ints
+            Determines the number of iterations performed to realign
+            scans within each run for each pass defined by the
+            ``speedup`` argument. For instance, setting ``speedup`` ==
+            (5,2) and ``loops`` == (5,1) means that 5 iterations are
+            performed in a first pass where scans are subsampled by an
+            isotropic factor 5, followed by one iteration where scans
+            are subsampled by a factor 2.
+        between_loops : None, int or sequence of ints
+            Similar to ``loops`` for between-run motion
+            estimation. Determines the number of iterations used to
+            realign scans across runs, a procedure similar to
+            within-run realignment that uses the mean images from each
+            run. If None, assumed to be the same as ``loops``.
+            The setting used in the experiments described in Roche,
+            IEEE TMI 2011, was: ``speedup`` = (5, 2), ``loops`` = (5,
+            1) and ``between_loops`` = (5, 1).
+        align_runs : bool
+            Determines whether between-run motion is estimated or
+            not. If False, the ``between_loops`` argument is ignored.
+        optimizer : str
+            Defines the optimization method. One of 'simplex',
+            'powell', 'cg', 'ncg', 'bfgs' and 'steepest'.
+        refscan : None or int
+            Defines the number of the scan used as the reference
+            coordinate system for each run. If None, a reference
+            coordinate system is defined internally that does not
+            correspond to any particular scan. Note that the
+            coordinate system associated with the first run is always
+            used for between-run motion estimation.
+        borders : sequence of ints
```

Comment:
Sorry - my question was bad - I think the order of parameters in the docstring does not match the signature (and didn't before this change either).  Is that right?